webpack.config.js

```diff
@@ -10,6 +10,16 @@ const OptimizeCSSAssetsPlugin = require("optimize-css-assets-webpack-plugin")
 module.exports = (env, argv) => {
     const isProduction = argv.mode === "production"
     return {
+        cache: {
+            type: "filesystem",
+            buildDependencies: {
+                config: [
+                    __filename,
+                    path.resolve(__dirname, "node_modules/.yarn-integrity"),
```

Comment:
you don't need that

```suggestion
```

This would invalidate the cache too often, since not all node_modules are buildDependencies. Some are normal dependencies (only imported in source files and not in config files).

Reply:
Oh wow, wouldn't have expected you to be looking at our PR - thank you so much!
From what I can tell webpack 5 works great for our code base, and the migration was surprisingly easy to do. So great work from you and your team on that one!

In this case, I simply wasn't sure what exactly should be included in the build dependencies, and I wanted to err on the more cautious side in case of doubt. I even saw [a test file](https://github.com/webpack/webpack/blob/c5add3175fd4120b0f5f8bba6dacf1420a0f0c2c/test/fixtures/buildDependencies/run.js) in your repo that has this exact config, so it might be worthwhile doing it this way.

So, maybe this is a hint that the documentation on persistent caching, while very good already, could be improved to include a few more examples of common files that should and should not be included in build dependencies.

But again, thank you for your great work and I'm glad (and surprised!) that you took a look at this PR